models/breast_density_classification/docs/README.md

```diff
@@ -0,0 +1,43 @@
+# Description
+A pre-trained model for breast-density classification.
+
+# Model Overview
+This model is trained using transfer learning on InceptionV3. The model weights were fine tuned using the Mayo Clinic Data. The details of training and data is outlined in https://arxiv.org/abs/2202.08238.
+
+The bundle does not support torchscript.
+
+# Input and Output Formats
+The input image should have the size [299, 299, 3]. For a dicom image which are single channel. The channel can be repeated 3 times.
+The output is an array with probabilities for each of the four class.
```

Comment:
HI @vikashg , could you please double confirm here? If the output is like this, `AsDiscreted` can be removed from `postprocessing` and we only need to keep the sigmoid.

Reply:
ok I can do that